Content/Bosses/SquidBoss/NPCs.SquidBoss.cs

```diff
@@ -829,6 +828,8 @@ public override void AI()
 
 				if (GlobalTimer > 240) //following unless using ink attack
 				{
+					NPC.dontTakeDamage = false;
```

Comment:
I don't think this guarantees solving it if it's related to this, we should add NPC.dontTakeDamage to send and receive additionalAI